### DISCUSSION TITLE
How can I auto log cypress test name?

### DISCUSSION THREAD

**Discussion by dushkin** [cypress-io/cypress#28717] [Questions and Help] [answered]
I’d like to be able to tell cypress to log each test name.

I know I can put inside each spec’s beforeEach() the following line:

`cy.log(‘Executing beforeEach() for: ' + Cypress.currentTest.title)`

But I don’t like the need to remember to add it to each beforeEach() hook implementation.

Is there a nicer way to achieve this goal?

**Comment by michallbanas** [comment, marked as answer]
Hey hey @dushkin 👋🏼 You don't need to remember it because you can add it globally in beforeEach loop to `support/command.js` file. 

```
beforeEach(() => {
    cy.log(`Executing beforeEach() for: ` + Cypress.currentTest.title)
  })
```

So this way it is added only once 🚀

**Reply by dushkin** [reply]
Toda Michal :)
Yes, I tried this way already and it works of course
